### DISCUSSION TITLE
Chapter14 ESlint error

### DISCUSSION THREAD

**Discussion by JawadTheGreat** [vercel/next.js#72881] [Help]
### Summary

In chapter 14(Improving accessibility) after running "pnpm lint" in terminal.
Expected output:
`✔ No ESLint warnings or errors`
Real output:
`./app/lib/actions.ts
33:12  Error: 'error' is defined but never used.  @typescript-eslint/no-unused-vars
57:12  Error: 'error' is defined but never used.  @typescript-eslint/no-unused-vars
72:12  Error: 'error' is defined but never used.  @typescript-eslint/no-unused-vars`

`./app/ui/customers/table.tsx
5:3  Error: 'CustomersTableType' is defined but never used.  @typescript-eslint/no-unused-vars`

`./app/ui/login-form.tsx
5:3  Error: 'ExclamationCircleIcon' is defined but never used.  @typescript-eslint/no-unused-vars`

**Comment by Mr-Meshky** [comment]
I think your problem is that you created a variable but did not use it for a while

If you want it to be in your code, it is a variable from
`// eslint-disable-next-line @typescript-eslint/no-unused-vars`
You can use it to fix it, otherwise it is better to delete it
